app/background.process.development.js

```diff
@@ -17,6 +17,9 @@ import { initAnon, initMock } from 'extensions/safe/network';
 import { setIPCStore } from 'extensions/safe/ffi/ipc';
 import manageRemoteCalls from './background.manageRemoteCalls';
 import sysUri from 'extensions/safe/ffi/sys_uri';
+import { loadExtensions }  from './extensions';
```

Comment:
@hunterlester let's rename this to something w/ `background` in it.

`loadBackgroundExtensions`, maybe. Or `initExtensionsInBackground`... (other ideas welcome!)

Reply:
hmmm, actually maybe something clearer for the extension, `onInitBgProcess` makes most sense